bindings/rust/s2n-tls-tokio/tests/shutdown.rs

```diff
@@ -391,16 +317,25 @@ async fn shutdown_with_tls_error_and_tcp_delay() -> Result<(), Box<dyn std::erro
         Pending
     })));
 
-    // Shutdown should complete with the correct error from s2n_shutdown
+    // Shutdown should complete with the correct error from s2n_shutdown_send
     let result = server.shutdown().await;
     let io_error = result.unwrap_err();
     let error: error::Error = io_error.try_into()?;
     // Any non-blocking read error is translated as "IOError"
     assert!(error.kind() == error::ErrorType::IOError);
 
-    // Even if s2n_shutdown fails, we need to close the underlying stream.
+    // Even if s2n_shutdown_send fails, we need to close the underlying stream.
     // Make sure we at least called our mock shutdown, consuming it.
     assert!(overrides.is_consumed());
 
```

Comment:
Do you think you could annotate the tests in this file with what specifically you're testing for? For some reason I'm having a hard time understanding what specifically you're trying to test, and it isn't clear from the test name.